NewForce-Cohort-9/bad-blood-brotherhood
# added files and folders for project start

## What Changed
1. Added file structure and paths for each country
2. That's it for this one

## How to Test – ***  rev-branch  ***
1.	Switch to my branch in terminal:
2.     git fetch --all
3.     git checkout rev-branch
4.     Check files and directories for proper location and spelling.
5.     Send me any errors you find and I will correct.
6.     If it looks correct, give me a thumbs up for confirmation. Thanks!
